pybamm/spatial_methods/spatial_method.py

```diff
@@ -291,12 +305,17 @@ def boundary_value_or_flux(self, symbol, discretised_child):
             The boundary value or flux symbol
         discretised_child : :class:`pybamm.StateVector`
             The discretised variable from which to calculate the boundary value
+        bcs : dict
+            The boundary conditions
```

Comment:
maybe add a sentence to say that, if available, these are used to improve the accuracy of the boundary value or flux